scripts/check_frontend_coverage_test.py

```diff
@@ -76,6 +76,14 @@ def test_get_stanzas_from_lcov_file(self):
             'LF:10\n'
             'LH:5\n'
             'end_of_record\n'
+            'SF:/opensource/oppia/invalid_file.ts\n'
```

Comment:
I don't think we should assume that the parent dir to be opensource

Reply:
Done.